spec/factories/academic_year.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :academic_year do
+    trait :current do
+      starts_on { Date.new(Date.current.year - 1, 9, 1) }
+      ends_on { Date.new(Date.current.year, 8, 31) }
```

Comment:
```suggestion
      starts_on { Date.parse("1 September #{Date.current.year - 1}") }
      ends_on { Date.parse("31 August #{Date.current.year}") }
```
This will increase readability.

I would also thing about doing something conditional when calculating these dates. So if you run these tests on 2nd September 2024, this will generate an academic year for 1st September 2023 - 31st August 2024, which might break some tests.

Reply:
I've tried running the specs with the following `around` hook.

```ruby
config.around do |example|
  Timecop.travel(Time.zone.local(2024, 9, 2)) do
    example.run
  end
end
```

All pass :)

Reply:
In which case I'm assuming your tests/code doesn't validate that the ClaimWindow dates must be within the AcademicYear `starts_on` and `ends_on` dates. 
So I don't think this will break your implementation.